ms2query_app.py

```diff
@@ -80,3 +81,9 @@
 
 query_spectrums = [post_process_s2v(spec) for spec in query_spectrums]
 library_spectrums = [post_process_s2v(spec) for spec in library_spectrums]
+
+# turn spectra into documents
+documents_query = [SpectrumDocument(spec, n_decimals=2) for spec in
+                   query_spectrums]
+documents_library = [SpectrumDocument(spec, n_decimals=2) for spec in
+                     library_spectrums]
```

Comment:
Seems like that is a good time to split something off again and move it to a separate function.
I anyway expect that the main app function will be a bit lengthy, so we should try to move everything we can out of it (also to allow for easier testing). Could be something like ``process_spectrums`` or any other name you can come up with that would combine the entire processing pipeline?

Reply:
Yes that seems indeed like a good idea.